docs/consensus/genesis.md

```diff
@@ -0,0 +1,37 @@
+# Genesis Document
+
+The genesis document contains a set of parameters that outline the initial state
+the [consensus layer] and its services.
+
+For more details about the actual genesis document's API, see
+[genesis API documentation].
+
+[consensus layer]: ../index.md
+[genesis API documentation]:
+  https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go/genesis/api
+
+## Genesis File
+
+A genesis file is a JSON document corresponding to a serialized genesis
+document.
+
+{% hint style="info" %}
+For a high-level overview of the genesis file, its various sections and
+parameters and the parameter values that will be used for Oasis Network Mainnet
+launch, see: [Genesis File Overview].
+{% endhint %}
+
+[Genesis File Overview]:
+  https://docs.oasis.dev/general/pre-mainnet/genesis-file
+
+### Canonical Form
+
+The *canonical* form of a genesis file is the pretty-printed JSON file with
```

Comment:
Maybe mention that this should not be confused with the canonical genesis document (used to derive the domain separation context) which is defined  ONLY as the canonical CBOR encoding of the genesis document.

Reply:
Good suggestion. I added a section on the domain separation context with links to other docs and a note like you suggested.

Please, take another look.